Dockerfile.pipelines

```diff
@@ -40,7 +55,10 @@ RUN if [ -f /usr/bin/gcov-tool-$gccvers ] ; then \
     fi
 
 # Install MPI
-RUN apt-get install -y -q mpich libmpich-dev
+RUN if [ $useOpenMP != "true" ] ; then \
```

Comment:
it seems that here we install MPI in all builds that won't use OpenMP, while [here](https://github.com/opesci/devito/blob/2de68cd5da086c1c0b888b6aca80586881527cfc/.travis.yml) the situation seems slightly different. At least the OpenMP build installs/uses MPI

Reply:
Ah, yes, I'd missed that the gcc5 build has neither - good spot. I'll fix that :-)